### ISSUE TITLE
Gnome 3.30 - Toggle overview moving windows down vertically

### ISSUE THREAD

**Issue by KGMaxey** [mpiannucci/gnome-shell-extended-gestures#60]
Distro: Fedora 29
Gnome 3.30

When you use a gesture to toggle into the overview, then use the gnome four finger gesture to change workspace, then use a gesture to toggle out of the overview, the windows in that workspace all jump down on the screen and out of view. Simply re-toggling the overview again fixes the issue, however it occasionally has some issues with displaying the correct currently selected window.

**Comment by flyingrub** [comment]
I can reproduce this on Arch linux and Gnome 3.30.

**Comment by Fallstar** [comment]
Also on Arch, this is happening all the time.

**Comment by mpiannucci** [comment]
Thats crazy, thanks for reporting. Not surprised, 3.30 brings a lot of changes that I still have to slog though. 
I am hopefully getting a new (personal) linux laptop soon to help me test quicker.

**Comment by Sangeppato** [comment]
I don't think this is an issue of this extension, since it happens as well with the stock gestures (pinch in with 3 fingers) and this extension completely removed
